frontend/scripts/tests/dashboard-element/fetch.saga.test.js

```diff
@@ -1,18 +1,14 @@
 import { initialState } from 'react-components/dashboard-element/dashboard-element.reducer';
 import { cancelled } from 'redux-saga/effects';
-import { fetchWithCancel } from 'utils/saga-utils';
-
+import fetchWithCancel from 'utils/fetchWithCancel';
```

Comment:
I prefer having this in the other file, to fix the mocks we can do what I did on the default columns branch